src/lab_sim/package.xml

```diff
@@ -14,6 +14,8 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <build_depend>examples_rclcpp_minimal_service</build_depend>
+  <build_depend>ros2action</build_depend>
```

Comment:
This is already present in the `moveit_studio_common` package - do we need it here? This package definitely transitively depends on that one

Reply:
I see now that you only had the action selected